front/app/containers/CustomPageShow/index.tsx

```diff
@@ -67,9 +70,19 @@ const CustomPageShow = () => {
     resourceId: !isNilOrError(page) ? page.id : null,
   });
 
-  if (isNilOrError(page) || isNilOrError(appConfiguration)) {
+  if (page === undefined || appConfiguration === undefined) {
```

Comment:
Added this to avoid the page flashing the 'Page not found' component while requests are pending